zoo-css-challenge/index.html

```diff
@@ -51,15 +52,15 @@ <h2>Bears</h2>
 		</section>
 
 		<section class="tigers">
-			<h2>Tiger</h2>
-			<h3>Top 5 Tiger Facts</h3>
-			<ol>
+			<h2>Tiger Facts</h2>
+			<!-- <h3>Top 5 Tiger Facts</h3> -->
```

Comment:
```suggestion
```
Software teams normally delete commented code, so we shouldn't commit commented code